include/bactria/Incident.hpp

```diff
@@ -0,0 +1,85 @@
+/* Copyright 2021 Jan Stephan
+ *
+ * Licensed under the EUPL, Version 1.2 or - as soon they will be approved by
+ * the European Commission - subsequent versions of the EUPL (the “Licence”).
+ * You may not use this work except in compliance with the Licence.
+ * You may obtain a copy of the Licence at:
+ *
+ *     http://ec.europa.eu/idabc/eupl.html
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the Licence is distributed on an “AS IS” basis, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  Licence permissions and limitations under the Licence.
```

Comment:
Are you aware of SPDX license identifiers? They make those headers a lot shorter.

Reply:
I am aware of SPDX. However, our current coding guidelines mandate the incorporation of the license header:

> each file must contain
> * copyright notice
>   * the corresponding license header, author(s) + current year ( if changed - and a commit of a file means a change )